ugvc/pipelines/filter_variants_pipeline.py

```diff
@@ -180,15 +180,13 @@ def run(argv: list[str]):
             )
             with pysam.VariantFile(args.output_file, mode="w", header=hdr) as outfile:
                 for i, rec in tqdm.tqdm(enumerate(infile)):
-                    pass_flag = False
                     if hmer_run[i]:
                         rec.info["HPOL_RUN"] = True
                     if args.model_file is not None:
                         if predictions[i] == "fp":
+                            if "PASS" in rec.filter.keys():
+                                del rec.filter["PASS"]
```

Comment:
can you explain the nature of these changes.
The context is not very clear to me.
What do you want to happenm and what happened before?